modules/game/src/main/Game.scala

```diff
@@ -926,7 +926,9 @@ object Game {
 
   val syntheticId = "synthetic"
 
-  val maxPlayingRealtime = 100 // plus 200 correspondence games
+  val maxPlayingRealtime = 100
```

Comment:
maxPlayingRealtime is now used in SimulApi only, but I agree having 100 games for a simul is already way enough